tests/unit_tests/testcases/app/test_utilsmisc.cpp

```diff
@@ -118,29 +118,30 @@ void TestUtilsMisc::testHtmlToMarkdown() {
 }
 
 void TestUtilsMisc::testParseTaskList() {
-    const auto listTag = QStringLiteral("<li style=\"list-style-type:square\">");
+    const auto taskListTag = QStringLiteral("<li class=\"task-list-item-checkbox\">");
+    const auto normalListTag = QStringLiteral("<li class=\"normal-list-item\">");
```

Comment:
I wonder what this is doing here if its not going to be used :grin:

Reply:
Waiting for its moment to shine! Oh well...